Controls/motor_powers.py

```diff
@@ -0,0 +1,63 @@
+import math
+import numpy as np
+from Simulations.DynamicSmoothedGUI import DynamicGUI
+
+"""
+Thoughts and considerations:
+    Should we send motor powers alone, or motor powers and time to apply those powers?
+    
+    Does C1C0 stop immediately once the motor powers have been applied for a certain amount of time,
+        or we do need to consider decelerating near the end point?
+
+    Will the input mappings tell us both distance and direction when a unit of power is applied to the motors?
+"""
+
+
+class MotorPower:
```

Comment:
Maybe mark this as unused in a comment so it is not confusing. But it is a good to keep in case we have to go back to this